app/controllers/api/v1/events_controller.rb

```diff
@@ -24,6 +24,16 @@ def create
 
       def update
         if event.update(event_params)
+          if params[:user_ids]
```

Comment:
Please move this logic to update service. Also do not forget to add current user id to `user_ids` array because it will destory EventUser records for all users whoose id do not present in user_ids array.

Reply:
This logic is no longer needed. It works automatically.